### PR TITLE
'View Output' button that directs to output channel when deleting blob folder

### LINKED ISSUE #426: [Suggestion]Add a "View Output" button to navigate to the Output window
**OS:** Windows_NT x64 10.0.17763
**Version**: 1.37.0-insider (user setup)
**Commit**: 5e08ee01b7daee99cdda2456f19d77e715fb8762
**Build Version:**[ 20190805.1](https://dev.azure.com/ms-azuretools/AzCode/_build/results?buildId=8478)
Regression: Not a Regression

**Repro Steps:**
1. Enable "Azure Storage: Enable View in File Explorer" in User Setting.
2. Click "Open in File Explorer..." for a blob container.
3. Make sure there is a folder containing a blob file which has special restrictions (has a snapshot or has a lease).
4. Right click this folder --> Select "Delete Permanently" --> Select "Delete" when asking whether to delete or not.
5. Make sure there is an error dialog.
image

**Actual:**
There is no button to navigate to Output window.

**Suggestion:**
Add a "View Output" button to navigate to the Output window directly like below screenshot:
image

### PR DESCRIPTION
**Pull request by monalgarg**
Fixes #426 , #427

### CODE REVIEW

File: src/azureStorageExplorer/blobContainers/BlobContainerFS.ts [hunk 1]
```diff
@@ -238,9 +238,17 @@ export class BlobContainerFS implements vscode.FileSystemProvider {
                     let errors: boolean = await this.deleteFolder(parsedUri, blobService);
 
                     if (errors) {
-                        vscode.window.showInformationMessage(`Errors occured when deleting ${parsedUri.filePath}. Please look at the output channel for more information.`);
                         // tslint:disable-next-line: no-multiline-string
                         ext.outputChannel.appendLine(`Please refresh the viewlet to see the changes made.`);
+
+                        const viewOutput: vscode.MessageItem = { title: 'View Output' };
+                        const errorMessage: string = `Errors occured when deleting ${parsedUri.filePath}. Please look at the output channel for more information.`;
+                        vscode.window.showInformationMessage(errorMessage, viewOutput).then(async (result: vscode.MessageItem | undefined) => {
```
Comment (EricJizbaMSFT) [inline src/azureStorageExplorer/blobContainers/BlobContainerFS.ts]: I suggest removing `Please look at the output channel for more information.` from the message and changing the button to `View Errors`

Comment (EricJizbaMSFT) [inline src/azureStorageExplorer/blobContainers/BlobContainerFS.ts]: This should be a warning message, not an info message

File: src/azureStorageExplorer/blobContainers/BlobContainerFS.ts [hunk 2]
```diff
@@ -238,9 +238,17 @@ export class BlobContainerFS implements vscode.FileSystemProvider {
                     let errors: boolean = await this.deleteFolder(parsedUri, blobService);
 
                     if (errors) {
-                        vscode.window.showInformationMessage(`Errors occured when deleting ${parsedUri.filePath}. Please look at the output channel for more information.`);
                         // tslint:disable-next-line: no-multiline-string
                         ext.outputChannel.appendLine(`Please refresh the viewlet to see the changes made.`);
+
+                        const viewOutput: vscode.MessageItem = { title: 'View Output' };
+                        const errorMessage: string = `Errors occured when deleting ${parsedUri.filePath}. Please look at the output channel for more information.`;
```
Comment (EricJizbaMSFT) [inline src/azureStorageExplorer/blobContainers/BlobContainerFS.ts]: As a general rule, you should always wrap a dynamic variable in quotes. Aka `"${parsedUri.filePath}"`

Comment (monalgarg) [reply src/azureStorageExplorer/blobContainers/BlobContainerFS.ts]: Like `Errors occured when deleting "${parsedUri.filePath}".`?

Comment (EricJizbaMSFT) [reply src/azureStorageExplorer/blobContainers/BlobContainerFS.ts]: Yeah

**Pull request merged**